pyigm/clustering/survey_2d2pcf.py

```diff
@@ -133,34 +178,71 @@ def calc_xi_gg(self, sigma=0, error=None):
         xi_gg and xi_gg_err_ls are also set internally
 
         """
+        if xi_type == 'galaxy-galaxy':
+            xa, xb = 'g', 'g'
+            auto = True
+        elif xi_type == 'absorber-galaxy':
+            xa, xb = 'a', 'g'
+            auto = False
+        else:
```

Comment:
for completitude, could you add a `xi_type == 'absorber-absorber'`?

Reply:
added